test/test_pytest_fixtures.py

```diff
@@ -51,3 +50,67 @@ def test_model_base(model_base):
     result.stdout.fnmatch_lines(
         ["*NotImplementedError*"]
     )
+
+
+class TestCreateEngineKwargs(object):
+
+    def test_default_create_engine_kwargs(self, create_engine_kwargs):
+        assert create_engine_kwargs == {}
```

Comment:
Oh, I misread the diff and thought your default was _not_ an empty dict (which it should be, for backwards compatibility). It's great as it is